### PR TITLE
Upgrade to Quarkus 2.13.3

### LINKED ISSUE #15032: Upgrade to Quarkus 2.13.3
### Description

The Quarkus team released 2.13.3 today, more precisely 3 hours ago with several bug fixes, including the update for CVEs on some dependencies like jackson-databind for example https://github.com/keycloak/keycloak/issues/14785.

We should consider upgrading before 20.0.0 if possible, now that the Quarkus team updated their dependencies.

### PR DESCRIPTION
**Pull request by abstractj**
Resolves #15032, #14785, #14833

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
         <infinispan.version>13.0.10.Final</infinispan.version>
         <infinispan.protostream.processor.version>4.4.1.Final</infinispan.protostream.processor.version>
         <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
-        <jackson.version>2.13.4</jackson.version>
+        <jackson.version>2.13.4.20221013</jackson.version>
```
Comment (pedroigor) [inline pom.xml:87]: Looks like we need here `2.13.4.2` as per https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13.

Did you use the script to upgrade Quarkus? If so, we should probably fix it to not get the version from the Jackson BOM.

Comment (abstractj) [reply pom.xml:87]: Now that you mentioned, I will definitely use the script. I have one small question. The reason why I updated jackson to `2.13.4.20221013` was to keep those versions in sync with Quarkus. See:  https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L86.

Do you think it is ok to update to `2.13.4.2`?

**Comment by martin-kanis** [comment]
We probably should't use `2.13.4.20221013`for now as it not available in maven central. If we gonna use `2.13.4.2` as Pedro mentioned we need to again split `Jackson` properties to `<jackson.version>` and `<jackson.databind.version>` because only `jackson-databind` was released in `2.13.4.2` version.

**Comment by pedroigor** [comment]
So I think it makes sense to split to avoid going through this issue in the future as the artifacts are released independently.

File: operator/pom.xml [hunk 2]
```diff
@@ -33,7 +33,6 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.operator.sdk.version>4.0.1</quarkus.operator.sdk.version>
-        <quarkus.version>2.12.2.Final</quarkus.version>
```
Comment (pedroigor) [inline operator/pom.xml:36]: @abstractj Thanks. I talked with @vmuzikar and it seems we can not align the versions between dist and operator. The operator is an extension from quarkiverse and as such has a different release cycle.

So the best is to keep this one as is.

**Review by martin-kanis — DISMISSED** [review]

**Review by vmuzikar — DISMISSED** [review]

**Review by pedroigor — APPROVED** [review]

**Review by pedroigor — CHANGES_REQUESTED** [review]
@abstractj Just realised that we should also update `wildfly-elytron` as per https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L110.

**Comment by abstractj** [comment]
> @abstractj Just realised that we should also update `wildfly-elytron` as per [https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L110](https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml?rgh-link-date=2022-10-21T19%3A12%3A55Z#L110).

Good catch @pedroigor, updated now. I haven't update the main pom because I believe it used by the legacy distribution, but please let me know if more changes are required.

**Comment by stianst** [comment]
> > @abstractj Just realised that we should also update `wildfly-elytron` as per [https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L110](https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml?rgh-link-date=2022-10-21T19%3A12%3A55Z#L110).
> 
> Good catch @pedroigor, updated now. I haven't update the main pom because I believe it used by the legacy distribution, but please let me know if more changes are required.

There is no legacy distribution anymore

**Review by vmuzikar — DISMISSED** [review]

**Review by martin-kanis — DISMISSED** [review]

**Review by pedroigor — DISMISSED** [review]

**Review by vmuzikar — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Pull request merged**